src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+module.exports = {
+  list: list
+};
+
+function list(settings, logger) {
+  fs.readdir(settings.pluginDir, function (err, files) {
+
+    var pluginFiles = files.filter(function (file) {
```

Comment:
We generally don't want to use `var` at all anymore. `const` will work in this case.

Another option is to skip a new variable entirely and just chain your `filter` and `forEach` together:

```js
files
.filter(function (file) {
  // ...
})
.forEach(function (pluginFile) {
  // ...
});
```